RadixWallet/Features/TransactionHistory/TransactionHistory+Reducer.swift

```diff
@@ -209,8 +209,8 @@ public struct TransactionHistory: Sendable, FeatureReducer {
 		switch internalAction {
 		case let .loadedFirstTransactionDate(date):
 			state.loading.isLoading = false
-			let lastDate: Date = .init(timeIntervalSinceNow: -10)
-			let firstDate = date ?? lastDate
+			let lastDate: Date = .now
+			let firstDate = date ?? lastDate.addingTimeInterval(-24 * 3600)
```

Comment:
This is just an arbitrary amount of time to get a non-empty time window when you have no transactions